app/controllers/concerns/foreman/controller/puppet/hosts_controller_extensions.rb

```diff
@@ -7,21 +7,17 @@ module Foreman::Controller::Puppet::HostsControllerExtensions
   MULTIPLE_EDIT_ACTIONS = %w(select_multiple_environment update_multiple_environment
                              select_multiple_puppet_proxy update_multiple_puppet_proxy
                              select_multiple_puppet_ca_proxy update_multiple_puppet_ca_proxy)
-  PUPPET_MULTIPLE_ACTIONS = %w(multiple_puppetrun update_multiple_puppetrun) + MULTIPLE_EDIT_ACTIONS
 
   included do
     add_smart_proxy_filters PUPPETMASTER_ACTIONS, :features => ['Puppet']
     alias_method :find_resource_for_puppet_host_extensions, :find_resource
     alias_method :ajax_request_for_puppet_host_extensions, :ajax_request
 
     before_action :ajax_request_for_puppet_host_extensions, :only => PUPPET_AJAX_REQUESTS
-    before_action :find_resource_for_puppet_host_extensions, :only => [:puppetrun]
     before_action :taxonomy_scope_for_puppet_host_extensions, :only => PUPPET_AJAX_REQUESTS
-    before_action :find_multiple_for_puppet_host_extensions, :only => PUPPET_MULTIPLE_ACTIONS
```

Comment:
this should stay with `MULTIPLE_EDIT_ACTIONS` as `PUPPET_MULTIPLE_ACTIONS` is a superset of those.